frontend/src/components/Home.js

```diff
@@ -1,223 +1,120 @@
-              data-accordion-target="#accordion-body-player"
-              aria-expanded="true"
-              aria-controls="accordion-body-player"
-            >
-              <span>Player Information: {store.data.player.full_name}</span>
-              <AccordionDownIcon />
-            </button>
-          </h2>
-          <div
-            id="accordion-body-player"
-            class="hidden"
-            aria-labelledby="accordion-heading-player"
-          >
-            <div class="py-5 border-b border-gray-200 dark:border-gray-700">
-              <Player player={store.data.player} />
-            </div>
-          </div>
-        </Show>
-        <Show when={store.data.wards}>
-          <For each={store.data.wards}>
-            {ward => (
-              <>
-                <h2 id={`accordion-heading-ward-${ward.id}`}>
-                  <button
-                    type="button"
-                    class="flex items-center justify-between w-full py-5 font-medium text-left text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400"
-                    data-accordion-target={`#accordion-body-ward-${ward.id}`}
-                    aria-expanded={store?.data?.player ? "false" : "true"}
-                    aria-controls={`accordion-body-ward-${ward.id}`}
-                  >
-                    <span>Player Information: {ward.full_name}</span>
-                    <AccordionDownIcon />
-                  </button>
-                </h2>
-                <div
-                  id={`accordion-body-ward-${ward.id}`}
-                  class="hidden"
-                  aria-labelledby={`accordion-heading-ward-${ward.id}`}
-                >
-                  <div class="py-5 border-b border-gray-200 dark:border-gray-700">
-                    <Player player={ward} />
+      <h2 class="text-center">By UPAI</h2>
+      <div class="mt-5">
+        <h3 class="text-2xl font-extrabold text-center">Features</h3>
+        <div
+          class={
+            "grid grid-cols-1 gap-5 md:grid-cols-2 lg:grid-cols-3 justify-items-center mt-5 "
+          }
+        >
+          <For each={features}>
+            {feature => (
+              <div class="block p-4 bg-white border border-blue-600 rounded-lg shadow dark:bg-gray-800 dark:border-blue-400 w-full">
+                <h5 class="mb-2 text-xl font-bold tracking-tight text-blue-600 dark:text-blue-400">
+                  {feature.title}
+                </h5>
+                <p class="font-normal text-gray-700 dark:text-gray-400">
+                  {feature.description}
+                </p>
+              </div>
+            )}
+          </For>
+        </div>
+      </div>
+      <div class="mt-10">
+        <h3 class="text-2xl font-extrabold text-center">
+          <mark class="px-2 text-white bg-blue-600 rounded dark:bg-blue-500">
```

Comment:
This is great. Can we link to the repository, here?

Reply:
Yep Let me do that

Reply:
Linked it now